apps/wallets/smart-wallet/next/README.md

```diff
@@ -1,40 +1,155 @@
+-   [![Crossmint](https://img.shields.io/badge/Crossmint-04CD6C?style=for-the-badge&logoColor=04CD6C&link=https://www.crossmint.com/)](https://www.crossmint.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+Follow these steps to set up your project locally and get it running. This guide provides clear instructions to help you get started quickly.
+
+### Prerequisites
 
 Before you begin, ensure you have the following installed:
 
 -   Node.js (v20.12 or later)
 -   npm (comes with Node.js)
 -   pnpm (`npm install -g pnpm`)
 
-After installing the prerequisites, clone the repository and install the dependencies:
+### Installation
 
-```bash
-git clone <repository-url>
-pnpm i
-```
+1. Obtain a free API Key from the Crossmint Console. Refer to [Get an API Key](https://docs.crossmint.com/wallets/smart-wallets/quickstart#2-get-an-api-key) from the Quickstart guide for detailed instructions.
+
+2. Clone the repo
+    ```sh
+    git clone https://github.com/Crossmint/crossmint-sdk.git
+    ```
+3. Install PNPM packages
+    ```sh
+    pnpm i
+    ```
+4. In the directory containing this README.md file, rename the `.env.example` file to `.env` and add your API key to the file.
+    ```bash
+    NEXT_PUBLIC_CROSSMINT_API_KEY="ENTER YOUR CROSSMINT API KEY";
+    ```
+5. Start the development server
+    ```sh
+    pnpm dev
+    ```
+6. Once you start the Next.js development server, the application can be found at `http://localhost:3000`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
 
-## Configuration
+Spin up your own instance of the Smart Wallets Demo in under 5 minutes! Below, you'll see how to add the provider and use the hook to integrate authentication into your application.
 
-Before running the application, you need to set up your environment variables:
+### Adding the Provider
 
-1. Where this README.md is contained, rename the `.env.example` file to `.env`.
-2. Open the `.env` file and fill in the necessary values as described below:
-    > The credentials can be found in our shared 1Pass (see admin for access).
+First, wrap your application with the `CrossmintAuthProvider` to provide authentication context to your components. This is typically done in the root of your app.
 
-```plaintext
-NEXT_PUBLIC_CROSSMINT_API_KEY=
+```tsx
+"use client";
+
+// Important: this ensures the client SDK only runs on the client
+import { CrossmintAuthProvider } from "@crossmint/client-sdk-react-ui";
+
+export default function App({ Component, pageProps }) {
+    return (
+        <CrossmintAuthProvider
+            apiKey="YOUR_CROSSMINT_API_KEY"
+            embeddedWallets={{
+                createOnLogin: "all-users",
+                defaultChain: "polygon-amoy",
+                type: "evm-smart-wallet",
+            }}
+        >
+            <Component {...pageProps} />
+        </CrossmintAuthProvider>
```

Comment:
This currently wouldn't work right?

Reply:
Since, for the time being, we require the environment prop too

Reply:
I've added the env prop in the example and made a reminder in the `environment` prop removal ticket to also remove it from the example.